api/controllers/CampaignBotController.js

```diff
@@ -239,39 +234,46 @@ class CampaignBotController {
   }
 
   /**
-   * Returns whether incoming request is an authorized clear cache command.
+   * Returns whether incoming request is the given command type.
    * @param {object} req
+   * @param {string} type
    * @return {bool}
    */
-  isCommandClearCache(req) {
-    const cmdClear = process.env.GAMBIT_CMD_CLEAR_CACHE;
-
-    if (!cmdClear) return false;
-
-    if (!req.user.role) return false;
+  isCommand(req, type) {
+    if (!type) {
+      return false;
+    }
 
-    if (req.user.role !== 'staff' && req.user.role !== 'admin') {
-      logger.warn(`${this.loggerPrefix(req)} unauthorized clear cache`);
+    const configName = `GAMBIT_CMD_${type.toUpperCase()}`;
+    const configValue = process.env[configName];
+    if (!configValue) {
+      logger.warn(`${this.loggerPrefix(req)} process.env.${configName} is undefined`);
 
       return false;
     }
 
-    const input = helpers.getFirstWord(req.incoming_message);
-    const isCommand = input && input.toUpperCase() === cmdClear.toUpperCase();
+    const result = this.parseCommand(req) === configValue.toUpperCase();
+
+    if (type === 'clear_cache') {
```

Comment:
Great catch @sergii-tkachenko -- this was firing when it wasn't supposed to because we do need that `result` -- https://github.com/DoSomething/gambit/pull/654/commits/41cfc05d239832cd6f04ae50379454b394ae07ba